versioned_docs/version-2.9/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md

```diff
@@ -231,4 +231,8 @@ To upgrade legacy multi-cluster apps:
 
 ## Limitations
 
-Dashboard apps or Rancher feature charts can't be installed using the Rancher CLI.
+- Dashboard apps or Rancher feature charts can't be installed using the Rancher CLI.
+
+When determining the most recent version to display for the **Upgradable** column on the **Apps > Installed Apps** page, rather than only considering versions of the Helm chart from the repository it was installed from, Rancher considers versions of the Helm chart from all repositories on the cluster.
+
+For example, you installed `cert-manager` v1.13.0 from repository A, where v1.14.0 is now the most recent version available. In this case, you expect **Upgradable** to display v1.14.0. However, if the cluster also has access to repository B where v1.15.0 of `cert-manager` is available, then **Upgradable** displays v1.15.0 even though the original installation used repository A.
```

Comment:
```suggestion
  When determining the most recent version to display for the **Upgradable** column on the **Apps > Installed Apps** page, rather than only considering versions of the Helm chart from the repository it was installed from, Rancher considers versions of the Helm chart from all repositories on the cluster.

  For example, you might install `cert-manager` v1.13.0 from repository A, where v1.14.0 is now the most recent version available. In this case, you expect **Upgradable** to display v1.14.0. However, if the cluster also has access to repository B where v1.15.0 of `cert-manager` is available, then **Upgradable** displays v1.15.0 even though the original installation used repository A.
```

Reply:
I'm probably missing the point, but what's the purpose of changing to `might install` here? Doesn't that alter the meaning?

Reply:
Trying to make it more speculative as it is an example. We could throw in an "if you installed" instead, or "For example, suppose you installed"

Reply:
> We could throw in an "if you installed" instead, or "For example, suppose you installed"

I think these make more sense (or maybe just adjusting the tense on the "might" portion). For upgrade scenarios, the precondition is that X is already installed (with the version having significance in this case) so making that part certain doesn't seem right logically.

Also, looks like we forgot the `-` before `When` since that's the start of the list item.